components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -144,7 +144,7 @@ class GeckoEngine(
      * Creates a new Gecko-based EngineView.
      */
     override fun createView(context: Context, attrs: AttributeSet?): EngineView {
-        return GeckoEngineView(context, attrs)
+        return GeckoEngineView(context, attrs, preferredColorScheme = settings.preferredColorScheme)
```

Comment:
See other comment. Let's use a method instead and invoke right after creating the instance?